transforms/utils.js

```diff
@@ -0,0 +1,109 @@
+// The following utils are borrowed from @simonihmig's
+// https://github.com/simonihmig/ember-test-helpers-codemod/blob/master/transforms/utils.js
+
+let _statementsToImport = new Set();
```

Comment:
I'd prefer to keep this inside the closure scope in the transform itself, and pass it in to the two methods. I'm generally very wary of "global mutatable state"...